firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -447,19 +447,15 @@ expected<percent_t> EtbController::getClosedLoopAutotune(percent_t target, perce
 		float kd = 0.08f * ku * m_tu;
 
 		// Every 5 cycles (of the throttle), cycle to the next value
-		if (m_autotuneCounter == 5) {
+		if (m_autotuneCounter >= 5) {
```

Comment:
the idea with the `>` was that it's more robust against an extra `++` slipped in somewhere, but it's not a big deal.

Reply:
That's why I added it ;)

Reply:
oh you ADDED it, apparently I can't read a diff